packages/calcite-components/src/utils/floating-ui.ts

```diff
@@ -124,7 +124,7 @@ export const placements = [
   "trailing-start"
 ] as const;
 
-export type LogicalPlacement = typeof placements[number];
+export type LogicalPlacement = (typeof placements)[number];
```

Comment:
Thanks for making these more explicit.

Reply:
I'm pretty sure these type formatting changes were done by `npm run lint`